src/store/weatherProviders/accuweather.js

```diff
@@ -0,0 +1,96 @@
+import * as utils from '../utils';
+
+export const saveCurrentPosition = async state => {
```

Comment:
Are you sure this method is saving current position? Because i don't think so

Reply:
Yeah, I forgot to change functions names